dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -166,6 +167,14 @@ open class Jpql : JpqlDsl {
         return Expressions.nullLiteral()
     }
 
+    /**
+     * Creates a datetime expression with the localDateTime.
+     */
```

Comment:
I think the comment below are better. I've adopted the comments from the contributor of `currentTime()`.

```
/**
 * Creates an expression that represents the local datetime.
 *
 * This is the same as ```LOCAL DATETIME```.
 */
```